trlx/pipeline/offline_pipeline.py

```diff
@@ -15,13 +15,27 @@ class PromptPipeline(BasePipeline):
     Tokenizes prompts, unless they are already tokenized, and truncates them to `max_prompt_length` from the right
     """
 
-    def __init__(self, prompts: List[str], max_prompt_length: int, tokenizer=None):
+    def __init__(self, prompts: List[str], max_prompt_length: int, tokenizer: PreTrainedTokenizer):
         super().__init__()
-        model_inputs = tokenizer(prompts, truncation=True, padding=False, max_length=max_prompt_length)
-        prompts = model_inputs["input_ids"]
+
+        # manually prepend bos token if not already present to match RL trainers tokenization
```

Comment:
This comment might be misleading since we no longer prepend BOS tokens from the trainers.

Reply:
Oh yeah, I'll change it. We still do add BOS and EOS for ILQL btw